examples/kafka-hub/hub/modules/persistence/persistence.bal

```diff
@@ -17,6 +17,7 @@
 import ballerina/websubhub;
 import kafkaHub.config;
 import kafkaHub.connections as conn;
+import ballerinax/kafka;
```

Comment:
it seems like the imports not ordered properly. Lets use the ballerina formatter

Reply:
Fixed with [358540c018e4393d49b3e38de7a53e6f0884660e](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/1051/commits/358540c018e4393d49b3e38de7a53e6f0884660e)